pkg/frontend/export.go

```diff
@@ -56,26 +51,20 @@ type ExportConfig struct {
 	Symbol      [][]byte
 	// default flush size
 	DefaultBufSize int64
-	OutputStr      []byte
-	LineSize       uint64
 
 	//file service & buffer for the line
-	UseFileService bool
 	writeParam
 	FileService fileservice.FileService
-	LineBuffer  *bytes.Buffer
 	Ctx         context.Context
 	AsyncReader *io.PipeReader
 	AsyncWriter *io.PipeWriter
 	AsyncGroup  *errgroup.Group
 	mrs         *MysqlResultSet
-	lineStr     []byte
```

Comment:
LineStr should be a local variable in exportDataFromResultSetToCSVFile. Removed in the Struct